.circleci/config.yml

```diff
@@ -198,6 +204,17 @@ jobs:
       - run:
           name: Fastlane
           command: cd ~/uport-mobile/ios && bundle exec fastlane buildTheApp
+      - run:
```

Comment:
Maybe we should run `bundle exec fastlane buildTheApp` only if appium tests pass?

Reply:
That's a good point.